README

```diff
@@ -287,8 +300,8 @@ either of these configuration steps for itself.
 
 FreeBSD
 -------
-In FreeBSD 10.0, netmap has been integrated into the core OS.  In order to use
-it, you must recompile your kernel with the line
+In FreeBSD 11.0 and higher, netmap has been integrated into the core OS.  In
```

Comment:
From what I've seen, it's compiled in with 11.0 by default, so the note about recompiling the kernel can go away.

Reply:
1.  Changed commit message titles to have "daq_netmap:" prefix.

2.  Removed info in README about recompiling FreeBSD kernel.